### ISSUE TITLE
While generating a response, screen readers are not announcing all the live progress messages in the chat consistently:A11y_Visual Studio Code_Github_Copilot_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#286725]
Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Feng.ms%2Fdocs%2Fcoreai%2Fdevdiv%2Fone-engineering-system-1es%2F1es-docs%2Fes-chat%2Findex&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221175207%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2FCAI%2520Accessibility%2520Support%2520Channel%3FgroupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221198042%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Foutlook.office.com%2Fbook%2Fmicrosoft2779007%2540microsoft.onmicrosoft.com%2Fs%2FbLeGBltMW0qI0WRLsTqx2A2%3Fismsaljsauthenabled&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221214666%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0).
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
 #A11yMAS; #A11yTCS; #Win32; #A11ySev2;#DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #FTP; #Screen Reader; #JAWS; #Element:chatboard; #STP; #NVDA; #ES Chat;

## Environment Details:
Application: VS Code
Visual studio code version: 1.108.0
OS: Windows 11 Enterprise 25H2
Build: 26100.2605
Screen reader
JAWS version: 2026
NVDA: 2025.3.2

### Pre-Requisites: 

1. Press Ctrl + Comma (,) to open the 'Settings' window and search for "Verbose Chat Progress Updates". 
2. Navigate to the "Verbose Chat Progress Updates" checkbox and press Enter to enable the setting. 

## Repro Steps
1. Turn on a screen reader and open the Visual Studio Code application. 
2. Now, navigate to the GitHub Copilot chat and type a prompt in the message edit field. 
3. Press Enter and observe the answer getting generated by an AI model. 
4. Verify whether the live progress messages are consistently read by the screen reader or not. 
5. Generate another response and wait for the live progress messages in the chat. 
6. Verify whether the live progress message is read or not. 

## Actual Experience.
While generating a response, screen readers are not announcing all the live progress messages in the chat consistently

## Expected Experience
When the AI model is generating a response, screen readers should announce the live progress messages in the GitHub Copilot chat as expected if the "Verbose Chat Progress Updates" is enabled. 

## User Impact:
Screen readers not announcing live progress messages consistently leave users uncertain about ongoing actions or updates. This lack of timely feedback can cause confusion, repeated actions, and a diminished overall experience for users who rely on assistive technologies. 

## Attachment
NVDA
https://github.com/user-attachments/assets/9cee4595-cc7e-412d-91d7-f444def9f6ad
JAWS
https://github.com/user-attachments/assets/a06ff663-0673-447f-9c1d-be349e2dedcc
